### PR TITLE
#7 - deploying website 

### LINKED ISSUE #7: Deploying website
I need a Github Action to deploy this website to our server.  I would wait until #2 and #6 wuold be implemented, as they'll probably change the structure of directories here.

### PR DESCRIPTION
**Pull request by Blusia**
add dev and prod things to deploy website to our server 
This should close #7

### CODE REVIEW

File: environment/prod/deployment/prod/.env.prod [hunk 1]
```diff
@@ -1,4 +1,4 @@
 APP_NAME=Legnicki Meetup Technologiczny
 ENVIRONMENT=prod
 
-LMT_HOST_NAME=`meetup.legnica.pl` , `www.meetup.legnica.pl`
+LMT_HOST_NAME=`meetup.legnica.pl`,`www.meetup.legnica.pl`
```
Comment (mtracz) [inline environment/prod/deployment/prod/.env.prod:4]: This redirect should be done in the domain DNS zone (www.meetup.legnica.pl -> meetup.legnica.pl).

Comment (Blusia) [reply environment/prod/deployment/prod/.env.prod:4]: So this is will be the right version?
`LMT_HOST_NAME=meetup.legnica.pl`

Comment (mtracz) [reply environment/prod/deployment/prod/.env.prod:4]: For me - yes. If we need to support `www.` prefix, then it should be done in the domain DNS zone as CNAME record.

**Review by EwelinaSkrzypacz — APPROVED** [review]
For me it works - on production e-mail are sent so approve.

File: .github/workflows/deploy-to-prod-init.yml [hunk 2]
```diff
@@ -15,10 +15,72 @@ jobs:
...
+      - name: set deployment project version
+        run: echo "DEPLOYMENT_PROJECT_VERSION=$(bash ./environment/prod/deployment/scripts/version.sh --long)" >> $GITHUB_ENV
+
+      - name: set up Docker Buildx
+        uses: docker/setup-buildx-action@v3.1.0
+
+      - name: login to GitHub Container Registry
+        uses: docker/login-action@v3
+        with:
+          registry: ghcr.io
```
Comment (mtracz) [inline .github/workflows/deploy-to-prod-init.yml:34]: Replace all occurrences `ghcr.io` with DOCKER_REGISTRY env.

**Review by EwelinaSkrzypacz — APPROVED** [review]

**Review by kamilpiech97 — APPROVED** [review]

**Review by mtracz — APPROVED** [review]

**Pull request merged**